repo/packages/C/cassandra/0/config.json

```diff
@@ -79,19 +79,19 @@
       "description": "CPU shares to allocate to each Cassandra Server Instance.",
       "type": "number",
       "minimum": 0.0,
-      "default": 2.0
+      "default": 0.1
     },
     "cassandra/resource/mem": {
       "description": "Memory (MB) to allocate to each Cassandra Server instance.",
       "type": "integer",
       "minimum": 0,
-      "default": 2048
+      "default": 768
     },
     "cassandra/resource/disk": {
       "description": "Disk (MB) to allocate to each Cassandra Server instance.",
       "type": "integer",
       "minimum": 0,
-      "default": 2048
+      "default": 16
```

Comment:
Wow, that's really small. But I guess we're not enabling disk isolation (yet), so it will minimize the scheduling/allocation impact and won't actually get enforced.